workers/social/lib/social/models/session.coffee

```diff
@@ -39,7 +39,7 @@ module.exports = class JSession extends Model
       impersonating     : Boolean
       # data holds arbitary transitive session based info, not necessarily
       # should be used. There won't be any kind of index on this field.
-      data              : Object
+      sessionData       : Object
```

Comment:
yup but using a field named `data` in the schema causing issues in Bongo https://github.com/koding/koding/pull/10651/commits/b9034363aca278f6bc364c05b3ac5ce3d04411de I'll later find a way to throw error once a schema has a field named `data`